telemetry/main.schema.json

```diff
@@ -478,7 +482,7 @@
                   "type": "string"
                 },
                 "version": {
-                  "type": "string"
+                  "type": ["string", "integer"]
```

Comment:
The [Telemetry Environment docs](https://gecko.readthedocs.io/en/latest/toolkit/components/telemetry/telemetry/data/environment.html) specify this as a string field, are you seeing cases where it's an integer?